torch/nn/utils/spectral_norm.py

```diff
@@ -21,11 +16,11 @@ def __init__(self, name='weight', n_power_iterations=1, eps=1e-12):
     def compute_weight(self, module):
         weight = module._parameters[self.name + '_org']
         u = module._buffers[self.name + '_u']
-        height, _cuda = weight.size(0), weight.is_cuda
+        height = weight.size(0)
         weight_mat = weight.view(height, -1)
         for _ in range(self.n_power_iterations):
-            v = l2normalize(torch.matmul(weight_mat.t(), u), self.eps)
-            u = l2normalize(torch.matmul(weight_mat, v), self.eps)
+            v = normalize(torch.matmul(weight_mat.t(), u), dim=0, eps=self.eps)
```

Comment:
Yes, `torch.matmul(weight_mat.t(), u)` returns 1d.
Since the arguments of `torch.nn.functional.normalize` are `input, p=2, dim=1, eps=1e-12`, I explicitly set `dim=0`.
Or, should I use `F.normalize`?

Reply:
using `torch.nn.functional.normalize` is fine then. I just wanted to make sure that we were not changing the behavior. Thanks!